src/i18n/locales/sl.json

```diff
@@ -0,0 +1,105 @@
+
+  "stop-button-title": "Prekini snemanje",
+  "upload-notification": "Trenutno nalaga ...",
+  "upload-settings-button-store": "Shrani nastavitve ",
+  "upload-settings-modal-header": "Opencast povezava",
+  "upload-settings-label-server-url": "URL strežnika ",
+  "upload-settings-label-username": "Uporabniško ime",
+  "upload-settings-label-password": "Geslo",
+  "upload-settings-invalid-url": "To ni veljaven URL",
+  "upload-settings-invalid-url-http-start": "URL povezava se mora začeti z 'https://' ali 'http://'",
+  "upload-settings-error-server-unreachable": "Strežnik, ki ste ga izbrali, ni dosegljiv. Možni vzroki: naslov spletnega strežnika, ki ste ga navedli, je napačen; vaša naprava je brez interrnetne povezave; strežnik Opencast je brez povezave; strežnik Opencast ni pravilno konfiguriran, ter ne dovoli CORS zahtevka iz programa Opencast Studio.",
+  "upload-settings-error-response-not-ok": "Strežnik je na našo zahtevo vrnil odgovor, ki ni v redu (Not OK). To verjetno pomeni, da strežnik, ki ste ga izbrai, ni veljaven strežnik Opencast.",
+  "upload-settings-error-invalid-response": "Strežnik je na našo zahtevo vrnil neveljaven odgovor. To verjetno pomeni, da strežnik, ki ste ga navedli, ni veljaven strežnik Opencast.",
+  "upload-settings-invalid-login-data": "Prijava v strežnik Opencast je neuspešna: uporabniško ime ali geslo, ki ste ga navedli, je neveljavno.",
+  "upload-settings-invalid-provided-login": "Strežnik sporoča, da omogoča samodejno prijavo, vendar je bila zahteva vrnjena kot neavtenticirana. To je ali notranja napaka programa Opencast Studio ali pa je vaš sistemski skrbnik napačno konfiguriral strežnik. Žal glede tega ne moremo pomagati :(",
+
+  "upload-time-a-few-seconds": "nekaj sekund",
+  "upload-time-a-minute": "minuta",
+  "upload-time-an-hour": "ura",
+  "upload-time-seconds": "sekunde",
+  "upload-time-minutes": "minute",
+  "upload-time-hours": "ure",
+  "upload-time-left": "{{time}} ostane",
+
+  "warning-https": "Večina brskalnikov ne dovoljuje zajema video posnetkov prek nešifrirane povezave (HTTP). Preklopite na HTTPS.",
+  "warning-recorder-not-supported": "Vaš brskalnik ne podpira snemanja video posnetkov.",
+  "warning-recorder-safari-hint": "Če uporabljate Safari, lahko v nastavitvah omogočite eksperimentalno funkcijo »MediaRecorder«. V Apple MacOS-u priporočamo prehod na Chrome ali Firefox.",
+  "warning-missing-connection-settings": "Povezava z Opencastom ni v celoti vzpostavljena: nalaganje je onemogočeno. Konfigurirajte povezavo v <1> nastavitvah </1> (posnetka ne boste izgubili).",
```

Comment:
`<1> nastavitvah </1>` should be `<1>nastavitvah</1>` (without space on the inside)

Reply:
```suggestion
  "warning-missing-connection-settings": "Povezava z Opencastom ni v celoti vzpostavljena: nalaganje je onemogočeno. Konfigurirajte povezavo v <1>nastavitvah</1> (posnetka ne boste izgubili).",
```